### ISSUE TITLE
Dates in links mangles the link

### ISSUE THREAD

**Issue by sphh** [cphyc/thunderbird_date#4]
I just received an email, where one embedded link contains a string, which looks like a date:
```
For further information, check the link<https://www.example.com/news/2/12032/open-positions>.
```

This is displayed in Thunderbird as:
For further information, check the [2/12032/open-positions"> link](https://www.example.com/news/%3Cabbr%20class=).

Please could you exempt links from being searched for dates?

Thunderbird: 115.4.2
Date detector: 0.0.7

**Comment by cphyc** [comment]
Good catch! Will work on this. As I mentioned in other issues, I have limited time to work on the project so any PR is more than welcome if this isn't fixed in the meantime.
